src/sentry/tasks/find_missing_codemappings.py

```diff
@@ -0,0 +1,91 @@
+import logging
+from ast import Tuple
+from datetime import timedelta
+from typing import List
+
+from django.utils import timezone
+
+from sentry.db.models.fields.node import NodeData
+from sentry.models import Project
+from sentry.models.group import Group
+from sentry.models.organization import Organization, OrganizationStatus
+from sentry.tasks.base import instrumented_task
+from sentry.utils.safe import get_path
+
+PREFERRED_GROUP_OWNERS = 1
+PREFERRED_GROUP_OWNER_AGE = timedelta(days=7)
+
+logger = logging.getLogger("tasks.commit_context")
+
+
+@instrumented_task(
+    name="sentry.tasks.find_missing_codemappings",
+    queue="find_missing_codemappings",
```

Comment:
Still need to register this, unless this task should be part of an existing queue?